### ISSUE TITLE
[FEED] consistent solution in host.py

### ISSUE THREAD

**Issue by msekania** [tribe29/ansible-collection-tribe29.checkmk#150]
Concerning the nasty host.py bug. Which as I see is now fixed with explicit call to "normalize_folder" function
line 299
```
current_folder = normalize_folder(current_folder)
```
actually, small change in line 140
```
        current_folder = "%s" % extensions.get("folder", "/")
```
instead of
```
        current_folder = "%s" % extensions.get("folder", "")
```
default folder should be “/” and not “”, might be more consistent

thanks, Michael

**Comment by lgetwan** [comment]
That's correct. I will take care of it.

**Comment by robin-tribe29** [comment]
Fixed in 993b4ce7a3957d881389f7f9216e828e7bebefc0.
